### ISSUE TITLE
Cant change quality settings youtube

### ISSUE THREAD

**Issue by sheez44** [sampotts/plyr#1207]
Here is an example: https://codepen.io/anon/pen/MPpMqa.

It totally ignores the set default value. I really need plyr to force play it in 1080p despite the speed of my connection.

**Comment by sampotts** [comment]
Yep. It's a bug with the YouTube API unfortunately. There's a plan to remove support for it as their API is useless. You _request_ a quality level but they never actually set it.
